ramp.yml

```diff
@@ -4,7 +4,8 @@ email: support@redislabs.com
 description: Serving tensors and executing deep learning graphs
 homepage: https://oss.redislabs.com/redisai/
 license: Redis Source Available License v1.0
-command_line_args: ""
+command_line_args: "BACKENDSPATH /var/opt/redislabs/modules/ai/{{NUMVER}}/deps"
+# command_line_args: "BACKENDSPATH /var/opt/redislabs/modules/ai/{{NUMVER}}/deps/backends"
```

Comment:
is this a leftover?

Reply:
LVGTM. only a small comment on a commented out line @rafie . Is it on purpose?

Reply:
Yes, I kind of left the option of having a one-contains-all-backends .tgz (which requires tweaking of the `BACKENDSPATH`).
We can probably drop this once it hits master as a comment.